scripts/lib/CIME/XML/env_batch.py

```diff
@@ -43,7 +43,7 @@ def set_value(self, item, value, subgroup=None, ignore_type=False):
                 elif value.count(":") == 2:
                     t_spec = "%H:%M:%S"
                 else:
-                    expect(False, "could not interpret format for wallclock time %s"%value)
+                    expect(False, "could not interpret format for wallclock time {}".format(value))
```

Comment:
I think you are right that both are currently supported, but if I recall the plan is not to support the old style.